src/bosh-director/spec/unit/addon/filter_spec.rb

```diff
@@ -213,7 +238,16 @@ module Addon
 
             it 'applies' do
               expect(addon_include.applies?(nil, nil)).to be(true)
-              expect(addon_include.applies?('anything', instance_group)).to be(true)
+              expect(addon_include.applies?(deployment, instance_group)).to be(true)
+            end
+          end
+
+          describe 'when addon is in deployment manifest and has team filter' do
+            let(:addon_level) { DEPLOYMENT_LEVEL }
+            let(:filter_hash) { {'teams' => ['team_5']} }
+
+            it 'does not do anything' do
```

Comment:
Is this actually correct? The spec says it does not apply but our expectation says that it should apply. One of these two is wrong.